components/admin/requests/permit-holder-information/Card.tsx

```diff
@@ -57,13 +66,30 @@ const Card: FC<Props> = props => {
     UpdatePermitHolderInformationRequest
   >(UPDATE_PERMIT_HOLDER_INFORMATION);
 
+  const [updateNewPermitHolderInformation] = useMutation<
+    UpdatePermitHolderInformationResponse,
+    UpdateNewPermitHolderInformationRequest
+  >(UPDATE_NEW_PERMIT_HOLDER_INFORMATION);
+
   if (!permitHolderInformation) {
     return null;
   }
 
   /** Handler for saving permit holder information */
   const handleSave = async (data: PermitHolderFormData) => {
-    await updatePermitHolderInformation({ variables: { input: { id: applicationId, ...data } } });
+    if (data.type === 'NEW') {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const { type, ...permitHolderData } = data;
```

Comment:
Let's move these two lines out of the if condition

Reply:
Moving them outside of the if condition results in a linting type error, will leave as is for now:
```
components/admin/requests/permit-holder-information/Card.tsx:84:22 - error TS2322: Type '{ firstName: string; middleName: Maybe<string>; lastName: string; receiveEmailUpdates: boolean; phone: string; email: Maybe<string>; addressLine1: string; addressLine2: Maybe<...>; ... 5 more ...; id: number; } | { ...; }' is not assignable to type 'UpdateNewApplicationGeneralInformationInput'.
  Type '{ firstName: string; middleName: Maybe<string>; lastName: string; receiveEmailUpdates: boolean; phone: string; email: Maybe<string>; addressLine1: string; addressLine2: Maybe<string>; city: string; postalCode: string; id: number; }' is missing the following properties from type 'UpdateNewApplicationGeneralInformationInput': dateOfBirth, gender, otherGender

84         variables: { input: { id: applicationId, ...permitHolderData } },
```